demisto_sdk/commands/validate/tests/PB_validators_test.py

```diff
@@ -327,6 +331,117 @@ def test_IsAskConditionHasUnhandledReplyOptionsValidator():
+                    "description": "",
+                    "playbookInputQuery": {"query": "", "queryEntity": "indicators"},
+                }
+            ],
+            False,
+        ],
+        pack_info={},
+    )
+    playbook.tasks = {
+        "0": TaskConfig(
+            **{
+                "id": "test fail task No1",
+                "taskid": "27b9c747-b883-4878-8b60-7f352098a631",
+                "type": "condition",
+                "message": {"replyOptions": ["yes"]},
+                "nexttasks": {"no": ["1"]},
+                "task": {"id": "27b9c747-b883-4878-8b60-7f352098a63c"},
+                "quietmode": 2,
+            }
+        ),
+        "1": TaskConfig(
+            **{
+                "id": "test fail task No2",
+                "taskid": "27b9c747-b883-4878-8b60-7f352098a631",
+                "type": "condition",
+                "message": {"replyOptions": ["yes"]},
+                "nexttasks": {"no": ["1"]},
+                "task": {"id": "27b9c747-b883-4878-8b60-7f352098a63c"},
+                "quietmode": 2,
+            }
+        ),
+    }
+    validator = IsTasksQuietModeValidator()
+    validate_res = validator.is_valid([playbook])
+    assert len(validate_res) == 1
+    assert (
+        (validate_res[0]).message
+        == "Playbook 'Detonate File - JoeSecurity V2' contains tasks that are not in quiet mode (quietmode: 2) The tasks names is: 'test fail task No1, test fail task No2'."
+    )
+    # fix the playbook
+    fix_playbook = validator.fix(playbook).content_object
+    # validate the fixing
+    assert len(validator.is_valid([fix_playbook])) == 0
+
+
+def test_IsTasksQuietModeValidator_pass_case():
+    """
+    Given:
+    - A valid playbook with tasks that "quietmode" field is 1
+
+    When:
+    - calling IsTasksQuietModeValidator.is_valid.
+    - calling IsTasksQuietModeValidator.fix
+
+    Then:
+    - The playbook is valid
+    - The playbook doesn't changed
+    """
+    playbook = create_playbook_object(
+        ["inputs", "quiet"],
+        [
+            [
+                {
+                    "value": {},
+                    "required": False,
+                    "description": "",
+                    "playbookInputQuery": {"query": "", "queryEntity": "indicators"},
+                }
+            ],
+            False,
+        ],
+    )
+    playbook.tasks = {
+        "0": TaskConfig(
+            **{
+                "id": "test task",
+                "taskid": "27b9c747-b883-4878-8b60-7f352098a631",
+                "type": "condition",
+                "message": {"replyOptions": ["yes"]},
+                "nexttasks": {"no": ["1"]},
+                "task": {"id": "27b9c747-b883-4878-8b60-7f352098a63c"},
+                "quietmode": 1,
+            }
+        )
+    }
+    validator = IsTasksQuietModeValidator()
+    assert len(validator.is_valid([playbook])) == 0
+    fix_playbook = validator.fix(playbook).content_object
+    assert fix_playbook == playbook
```

Comment:
Not sure what you tried to do here?
If line 441 return 0 then what are we fixing?